SPECS/ibmtpm/ibmtpm.spec

```diff
@@ -0,0 +1,78 @@
+%define ibmversion  1682
+
+Summary:        This project is an implementation of the TCG TPM 2.0 specification.
+Name:           ibmtpm
+Version:        0.%{ibmversion}
```

Comment:
Looks like Photon is going with the direct version, e.g. `1682`, maybe there should be a comment here so future updaters understand they should move the Photon version up to the define at the top

Reply:
i can do that ... is this an ok approach when the versioning is so different from our versioning?

Reply:
added comment at top

Reply:
Well, we can certainly use whatever version we want, so I don't think there is specifically anything wrong with your approach, although with that said, if you don't have any actual reason to diverge from Photon's versioning, it'll certainly be easier later to just use what they decided on. Personally I probably would just stick with the upstream packager's version, but either way should be fine.